### PR TITLE
Next release

### PR DESCRIPTION
**Pull request by felipecrs**
- [x] Replace references to old repository name
- [x] Bump version
- [x] Fix changelog

### CODE REVIEW

**Comment by felipecrs** [comment]
Do we have everything we need for the next release or should we wait for something?

**Comment by nanosonde** [comment]
The asterisk integration won‘t work anymore.
It will not show the correct extension state.
Also the device[„tech“] is broken. We now only have pjsip endpoints which uses another event „EndpointList“ and has other fields in it, than the the old SIP peer stuff.

**Comment by nanosonde** [comment]
Opened this issue https://github.com/TECH7Fox/asterisk-hass-integration/issues/49

**Comment by TECH7Fox** [comment]
Yes, the card and integration work separate. The integration is far from done so I will some work more on it.

As long as the card and add-on work together out of the box, I think we can merge.

**Review by TECH7Fox — DISMISSED** [review]

**Comment by nanosonde** [comment]
Just found a minor issue while reading the logs:
```
[Mar 15 09:03:55] ERROR[476]: loader.c:2508 load_modules: Error loading module 'format_ogg_vorbis.so': libvorbisenc.so.2: cannot open shared object file: No such file or directory
```

Besides the debian package `libvorbis0a` we also have to add `libvorbisenc2` and `libvorbisfile3` as well to the Dockerfile.

**Comment by nanosonde** [comment]
See PR https://github.com/TECH7Fox/asterisk-hass-addons/pull/120/

**Comment by nanosonde** [comment]
And a little polish https://github.com/TECH7Fox/asterisk-hass-addons/pull/121

**Comment by nanosonde** [comment]
I have tested an amd64 images with this PR merged with PR120 and PR121. 
It works good now. Without any error messages.

**Comment by felipecrs** [comment]
I think we should bump version to 2.0.0.

**Comment by TECH7Fox** [comment]
I agree. This is a major update.

**Comment by felipecrs** [comment]
@TECH7Fox please merge if it's good for you. It's good for me.

**Comment by nanosonde** [comment]
@TECH7Fox @felipecrs 
Let's freeze this branch now and I will manually test this again one more time.
And if it's good let's release it....

**Comment by nanosonde** [comment]
@TECH7Fox I have tested a local build for amd64 of this current state and it is working on my production system using latest HA-OS, HA core and HA supervisor.

The build was done by using the home assistant addon builder.

So 🟢 from my side!

File: asterisk/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,28 @@
...
+
+Lots of issues were fixed by the above.
+
+### Upgrade guide
+
+It's strongly recommended to erase your existing Asterisk configuration before upgrading.
+
+1. Move any customization you have done in `/config/asterisk/` to somewhere else.
+2. Delete the `/config/asterisk` folder.
+3. Restore your customizations to the `/etc/config` folder if you have any.
```
Comment (TECH7Fox) [inline asterisk/CHANGELOG.md:22]: What do you mean with this? Did you mean /config/asterisk?

And maybe add "start the add-on to generate the new config files"?

File: repository.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # https://developers.home-assistant.io/docs/add-ons/repository#repository-configuration
 name: Asterisk Add-on
-url: "https://github.com/TECH7Fox/Asterisk-add-on"
-maintainer: Jordy Kuhne <jordy.kuhn@gmail.com>
\ No newline at end of file
+url: "https://github.com/TECH7Fox/asterisk-hass-addons"
+maintainer: Jordy Kuhne <jordy.kuhn@gmail.com>
```
Comment (TECH7Fox) [inline repository.yaml:4]: I now see that I missed a 'e'.

```suggestion
maintainer: Jordy Kuhne <jordy.kuhne@gmail.com>
```

**Review by TECH7Fox — CHANGES_REQUESTED** [review]

**Comment by felipecrs** [comment]
@TECH7Fox you need to approve the PR before I can merge.

**Review by TECH7Fox — APPROVED** [review]

**Comment by TECH7Fox** [comment]
Sorry for the wait, currently at work.

**Comment by felipecrs** [comment]
> Sorry for the wait, currently at work.

No problem :)

**Comment by felipecrs** [comment]
@nanosonde huuge thanks for your contributions!

**Comment by TECH7Fox** [comment]
Thanks @nanosonde and @felipecrs for your work! This is awesome. :)

@nanosonde, are you in the discord server?

**Comment by nanosonde** [comment]
> Thanks @nanosonde and @felipecrs for your work! This is awesome. :)
> 
> @nanosonde, are you in the discord server?

Which dscord server? Never used such a thing. ;-)

BTW: I do not see any new release or new packages for 2.0.0. I did not look at the Github actions, but isn't this done automatically for this repo?

**Comment by TECH7Fox** [comment]
@nanosonde here is a link: https://discord.gg/qxnDtHbwuD

**Pull request merged**